dwave_networkx/drawing/qubit_layout.py

```diff
@@ -143,47 +128,37 @@ def node_color(v):
         # the range of the color map is shared for nodes/edges and is symmetric
         # around 0.
         vmag = max(max(abs(c) for c in node_color), max(abs(c) for c in edge_color))
-        if vmin is None:
-            vmin = -1 * vmag
-
-        if vmax is None:
-            vmax = vmag
 
-        if edge_vmin is None:
-            edge_vmin = -1 * vmag
+        # since we're applying the colormap here, matplotlib throws warnings if
+        # we provide these arguments and it doesn't use them.
+        cmap = kwargs.pop('cmap', None) or plt.get_cmap('coolwarm')
+        vmin = kwargs.pop('vmin', None) or -1 * vmag
```

Comment:
`or` might not be what you want here. E.g. if user explicitly sets `vmin=0`, you override it to `-vmag`.

Reply:
When I say "here", I mean in this line, and a few other lines below.